addons/website/controllers/main.py

```diff
@@ -223,6 +223,16 @@ def website_info(self, **kwargs):
         }
         return request.render('website.website_info', values)
 
+    @http.route(['/website/configurator/<int:step>'], type='http', auth="user", website=True)
+    def website_configurator(self, step=1, wid=None, **kwargs):
```

Comment:
- that `step` parameter looks weird. why do you need different URLs for the same result? there are other ways to keep track of the state on the client-side without introducing "dummy" controller params, no?
- wtf is `wid`?

Reply:
wid is not needed anymore. I forgot to clean it from a previous iteration.
I use the owl router. The step is used to easily determine which component should be displayed. https://github.com/odoo/owl/blob/master/doc/reference/router.md